lib/engine/part/path.rb

```diff
@@ -51,10 +50,36 @@ def <=>(other)
       end
 
       def <=(other)
+        other_ends = other.ends
+        ends.all? { |t| other_ends.any? { |o| t <= o } }
+      end
+
+      def match?(other)
```

Comment:
do we need the match method? should the new <= method just replace it?

Reply:
We still need it to make sure we haven't added unreachable track.

Reply:
It's only a problem when upgrading Lawson tiles.

Reply:
why doesn't <= work when upgrading lawson?

Reply:
Fixed it so that it does now.